tests/test_freqtradebot.py

```diff
@@ -416,8 +416,8 @@ def test_enter_positions_global_pairlock(default_conf, ticker, limit_buy_order,
     assert log_has_re(message, caplog)
```

Comment:
My caplog has `No currency pair in active pair whitelist, but checking to sell open trades.` and I can't see where I could dit the whitelist for this test